src/development/LocalTest/Controllers/Authentication/AuthenticationController.cs

```diff
@@ -45,7 +45,7 @@ public ActionResult RefreshJWTCookie()
             ClaimsPrincipal principal = HttpContext.User;
             logger.LogInformation("Refreshing token....");
 
-            string token = jwtHandler.GenerateToken(principal, new TimeSpan(0, Convert.ToInt32(generalSettings.GetJwtCookieValidityTime), 0));
+            string token = jwtHandler.GenerateToken(principal, new TimeSpan(0, Convert.ToInt32(generalSettings.GetJwtCookieValidityTime), 0)).Result;
```

Comment:
This should at some point be converted into proper async / await, however one can do it at a later point.

Reply:
agreed